src/tests/intg/test_kcm.py

```diff
@@ -178,6 +178,12 @@ def setup_for_kcm_sec(request, kdc_instance):
     Just set up the local provider for tests and enable the KCM
     responder
     """
+    sec_resp_path = os.path.join(config.LIBEXEC_PATH, "sssd", "sssd_secrets")
+    if not os.access(sec_resp_path, os.X_OK):
+        # It would be cleaner to use pytest.mark.skipif on the package level
+        # but upstream insists on supporting RHEL-6.
```

Comment:
We should **not** support EL-6 anymore.

Reply:
Yes, but changing the tests to use new pytest features is outside the scope of this PR. There are more things I would like to do like having parametrized fixtures etc.

Would you prefer if I reword the comments to say that this should be changed when RHEL-6 support is dropped? And/or file a ticket to stop using old pytest features?

Reply:
Yes, please, then we at least keep track of it.

Reply:
https://pagure.io/SSSD/sssd/issue/3803